.guthub/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: Test
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn install --frozen-lockfile
+      - run: yarn test
+      - run: node_modules/.bin/codecov
```

Comment:
```suggestion
      - run: yarn codecov
```

probably works?

Reply:
yeah i wasn't sure, the docs are, of course, terrible.